### PR TITLE
Remove Newspack UI from Ads Admin page

### PR DESCRIPTION
**Pull request by adekbadek**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Newspack UI will be used in the Newspack Plugin

### How to test the changes in this Pull Request:

1. Visit Newsletters -> Ads page 
2. Observe a regular WP post list UI
3. Create an Ad, verify add creation/editiing/removing works as before

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: includes/class-newspack-newsletters-ads.php [hunk 1]
```diff
@@ -140,8 +95,26 @@ public static function register_ads_cpt() {
...
+			'add_new_item'       => __( 'Add New Newsletter Ad', 'newspack-newsletters' ),
+			'new_item'           => __( 'New Newsletter Ad', 'newspack-newsletters' ),
+			'edit_item'          => __( 'Edit Newsletter Ad', 'newspack-newsletters' ),
+			'view_item'          => __( 'View Newsletter Ad', 'newspack-newsletters' ),
+			'all_items'          => __( 'All Newsletter Ads', 'newspack-newsletters' ),
+			'search_items'       => __( 'Search Newsletter Ads', 'newspack-newsletters' ),
+			'parent_item_colon'  => __( 'Parent Newsletter Ads:', 'newspack-newsletters' ),
+			'not_found'          => __( 'No Newsletter Ads found.', 'newspack-newsletters' ),
+			'not_found_in_trash' => __( 'No Newsletter Ads found in Trash.', 'newspack-newsletters' ),
+		];
```
Comment (thomasguillot) [inline includes/class-newspack-newsletters-ads.php:113]: Could you add more labels so we avoid seeing something like my screenshot bellow when publishing/updating an ad.

```
'items_list' => __( 'Newsletter Ads list', 'newspack-newsletters' ),
'item_published' => __( 'Newsletter Ad published', 'newspack-newsletters' ),
'item_published_privately' => __( 'Newsletter Ad published privately', 'newspack-newsletters' ),
'item_reverted_to_draft' => __( 'Newsletter Ad reverted to draft', 'newspack-newsletters' ),
'item_scheduled' => __( 'Newsletter Ad scheduled', 'newspack-newsletters' ),
'item_updated' => __( 'Newsletter Ad updated', 'newspack-newsletters' ),
```

Screenshot 2020-08-26 at 12 14 26

File: includes/class-newspack-newsletters-ads.php [hunk 2]
```diff
@@ -140,8 +95,26 @@ public static function register_ads_cpt() {
...
+
 		$cpt_args = [
 			'public'       => false,
+			'labels'       => $labels,
 			'show_ui'      => true,
 			'show_in_menu' => false,
```
Comment (thomasguillot) [inline includes/class-newspack-newsletters-ads.php:119]: Any chance we can have the menu item above Settings?

Screenshot 2020-08-26 at 12 20 52

**Review by thomasguillot — CHANGES_REQUESTED** [review]
This looks great. Just a couple of things 😊

File: includes/class-newspack-newsletters-ads.php (hunk 2 above)
Comment (adekbadek) [reply includes/class-newspack-newsletters-ads.php:127]: Done in ab5ac78e350251c0323bc1ee5dee805f4d5dd6aa

File: includes/class-newspack-newsletters-ads.php (hunk 1 above)
Comment (adekbadek) [reply includes/class-newspack-newsletters-ads.php:121]: Done in ab5ac78e350251c0323bc1ee5dee805f4d5dd6aa

**Review by thomasguillot — APPROVED** [review]
Nice 🚢

**Pull request merged**